iothub/service/src/Feedback/Models/FeedbackRecord.cs

```diff
@@ -20,7 +20,7 @@ public class FeedbackRecord
         /// <remarks>
         /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
         /// </remarks>
-        [JsonProperty("originalMessageId", DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
q - Is this worth retaining, only because NewtonSoft does let us do this, and it might save someone some bandwidth?

Reply:
Bandwidth is an interesting reason.

I was thinking more about, across the board and not specifically on this property, enabling users to remove a property by nulling it out. We should be including their nulls. But how do we distinguish between null being default and a model formerly having a value that is now null?

Reply:
iirc the only place where service supports removing properties based on "null" value are the desired and reported "properties" within a twin. I don't think other model properties support this behavior. So, for the rest of the properties, as long as the property is optional excluding it should be acceptable.